server/routes/notification.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+
+const {
+    validateMakeNotification, 
+    validateMarkReadNotification,
+    validateNotification,
+} = require('../middleware/notification');
+
+const { 
+    makeNotification, 
+    markReadNotification, 
+    getAllNotifications, 
+    getUnreadNotifications 
+} = require('../controllers/notification'); 
+
+router.route("/")
+    .all(protect)
+    .get(getAllNotifications)
+    .post(validateMakeNotification, validateNotification, makeNotification);
+router.route("/mark-read")
```

Comment:
Not a huge fan of this route name `mark-read`. Wonder if we can do a different name?